static/src/javascripts/projects/commercial/modules/hosted/youtube.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import nextVideoAutoplay from 'commercial/modules/hosted/next-video-autoplay';
+import { initYoutubePlayer } from 'common/modules/atoms/youtube-player';
+import tracking from 'common/modules/atoms/youtube-tracking';
+import detect from 'lib/detect';
+import mediator from 'lib/mediator';
+
+type videoPlayerComponent = { getCurrentTime: () => number };
+
+const EVENTSFIRED = [];
+
+const isDesktop = (): boolean => detect.isBreakpoint({ min: 'desktop' });
+
+const sendPercentageCompleteEvents = (
+    atomId: string,
+    youtubePlayer: videoPlayerComponent,
+    playerTotalTime: number
+): void => {
+    const quartile = playerTotalTime / 4;
+    const playbackEvents = {
+        '25': quartile,
+        '50': quartile * 2,
+        '75': quartile * 3,
+    };
+
+    Object.keys(playbackEvents).forEach(key => {
+        const value = playbackEvents[key];
```

Comment:
You may want to use [`Object.entries`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) for extra awesomeness 😅